frontend/components/usage.js

```diff
@@ -264,6 +264,16 @@ export default function Usage({ baseUrl }) {
           &gt;&amp;color=&lt;COLOR&gt;&amp;prefix=&lt;PREFIX&gt;&amp;suffix=&lt;SUFFIX&gt;
         </StyledCode>
       </p>
+      <p>
+        <StyledCode>
+          {baseUrl}
+          /badge/dynamic/html.svg?url=&lt;URL&gt;&amp;label=&lt;LABEL&gt;&amp;query=&lt;
+          <a href="http://xpather.com" target="_BLANK" title="XPath syntax">
```

Comment:
Lowercasing an xpath is not in my opinion an option, because for example attribute _values_ are case sensitive.

Will add a note to the xpather link title, not sure if it should be more prominent than that. Maybe a small "fine print" below the html styledcode link?